<ui version="4.0" >
 <class>PenEditorForm</class>
 <widget class="QWidget" name="PenEditorForm" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>300</width>
    <height>391</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Pen properties</string>
  </property>
  <layout class="QVBoxLayout" >
   <property name="spacing" >
    <number>6</number>
   </property>
   <property name="margin" >
    <number>9</number>
   </property>
   <item>
    <widget class="QGroupBox" name="previewBox" >
     <property name="title" >
      <string>Preview</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="lineStyleBox" >
     <property name="title" >
      <string>Line style</string>
     </property>
     <layout class="QHBoxLayout" >
      <property name="spacing" >
       <number>6</number>
      </property>
      <property name="margin" >
       <number>9</number>
      </property>
      <item>
       <layout class="QVBoxLayout" >
        <property name="spacing" >
         <number>6</number>
        </property>
        <property name="margin" >
         <number>0</number>
        </property>
        <item>
         <widget class="QRadioButton" name="noneLineStyleBtn" >
          <property name="text" >
           <string>None</string>
          </property>
          <property name="checked" >
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="solidLineStyleBtn" >
          <property name="text" >
           <string>Solid</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="dashLineStyleBtn" >
          <property name="text" >
           <string>Dash</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" >
        <property name="spacing" >
         <number>6</number>
        </property>
        <property name="margin" >
         <number>0</number>
        </property>
        <item>
         <widget class="QRadioButton" name="dotLineStyleBtn" >
          <property name="text" >
           <string>Dot</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="dashDotLineStyleBtn" >
          <property name="text" >
           <string>Dash &amp;&amp; dot</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="dashDotDotLineStyleBtn" >
          <property name="text" >
           <string>Dash &amp;&amp; double dot</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" >
     <property name="spacing" >
      <number>6</number>
     </property>
     <property name="margin" >
      <number>0</number>
     </property>
     <item>
      <widget class="QLabel" name="label" >
       <property name="text" >
        <string>Line width</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="lineWidthCombo" >
       <property name="editable" >
        <bool>true</bool>
       </property>
       <property name="currentIndex" >
        <number>1</number>
       </property>
       <property name="maxVisibleItems" >
        <number>12</number>
       </property>
       <property name="minimumContentsLength" >
        <number>1</number>
       </property>
       <item>
        <property name="text" >
         <string>1</string>
        </property>
       </item>
       <item>
        <property name="text" >
         <string>2</string>
        </property>
       </item>
       <item>
        <property name="text" >
         <string>3</string>
        </property>
       </item>
       <item>
        <property name="text" >
         <string>4</string>
        </property>
       </item>
       <item>
        <property name="text" >
         <string>5</string>
        </property>
       </item>
       <item>
        <property name="text" >
         <string>6</string>
        </property>
       </item>
       <item>
        <property name="text" >
         <string>7</string>
        </property>
       </item>
       <item>
        <property name="text" >
         <string>8</string>
        </property>
       </item>
       <item>
        <property name="text" >
         <string>9</string>
        </property>
       </item>
       <item>
        <property name="text" >
         <string>10</string>
        </property>
       </item>
       <item>
        <property name="text" >
         <string>11</string>
        </property>
       </item>
       <item>
        <property name="text" >
         <string>12</string>
        </property>
       </item>
       <item>
        <property name="text" >
         <string>13</string>
        </property>
       </item>
       <item>
        <property name="text" >
         <string>14</string>
        </property>
       </item>
       <item>
        <property name="text" >
         <string>15</string>
        </property>
       </item>
       <item>
        <property name="text" >
         <string>16</string>
        </property>
       </item>
       <item>
        <property name="text" >
         <string>17</string>
        </property>
       </item>
       <item>
        <property name="text" >
         <string>18</string>
        </property>
       </item>
       <item>
        <property name="text" >
         <string>19</string>
        </property>
       </item>
       <item>
        <property name="text" >
         <string>20</string>
        </property>
       </item>
       <item>
        <property name="text" >
         <string>21</string>
        </property>
       </item>
       <item>
        <property name="text" >
         <string>22</string>
        </property>
       </item>
       <item>
        <property name="text" >
         <string>23</string>
        </property>
       </item>
       <item>
        <property name="text" >
         <string>24</string>
        </property>
       </item>
       <item>
        <property name="text" >
         <string>25</string>
        </property>
       </item>
       <item>
        <property name="text" >
         <string>26</string>
        </property>
       </item>
       <item>
        <property name="text" >
         <string>27</string>
        </property>
       </item>
       <item>
        <property name="text" >
         <string>28</string>
        </property>
       </item>
       <item>
        <property name="text" >
         <string>29</string>
        </property>
       </item>
       <item>
        <property name="text" >
         <string>30</string>
        </property>
       </item>
       <item>
        <property name="text" >
         <string>35</string>
        </property>
       </item>
       <item>
        <property name="text" >
         <string>40</string>
        </property>
       </item>
       <item>
        <property name="text" >
         <string>45</string>
        </property>
       </item>
       <item>
        <property name="text" >
         <string>50</string>
        </property>
       </item>
       <item>
        <property name="text" >
         <string>55</string>
        </property>
       </item>
       <item>
        <property name="text" >
         <string>60</string>
        </property>
       </item>
       <item>
        <property name="text" >
         <string>70</string>
        </property>
       </item>
       <item>
        <property name="text" >
         <string>80</string>
        </property>
       </item>
       <item>
        <property name="text" >
         <string>90</string>
        </property>
       </item>
       <item>
        <property name="text" >
         <string>100</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" >
     <property name="spacing" >
      <number>6</number>
     </property>
     <property name="margin" >
      <number>0</number>
     </property>
     <item>
      <widget class="QGroupBox" name="capStyleBox" >
       <property name="title" >
        <string>Cap style</string>
       </property>
       <layout class="QHBoxLayout" >
        <property name="spacing" >
         <number>6</number>
        </property>
        <property name="margin" >
         <number>9</number>
        </property>
        <item>
         <layout class="QVBoxLayout" >
          <property name="spacing" >
           <number>6</number>
          </property>
          <property name="margin" >
           <number>0</number>
          </property>
          <item>
           <widget class="QRadioButton" name="squareCapStyleBtn" >
            <property name="text" >
             <string>Square</string>
            </property>
            <property name="checked" >
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="flatCapStyleBtn" >
            <property name="text" >
             <string>Flat</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="roundCapStyleBtn" >
            <property name="text" >
             <string>Round</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="joinStyleBox" >
       <property name="title" >
        <string>Join style</string>
       </property>
       <layout class="QHBoxLayout" >
        <property name="spacing" >
         <number>6</number>
        </property>
        <property name="margin" >
         <number>9</number>
        </property>
        <item>
         <layout class="QVBoxLayout" >
          <property name="spacing" >
           <number>6</number>
          </property>
          <property name="margin" >
           <number>0</number>
          </property>
          <item>
           <widget class="QRadioButton" name="bevelJoinStyleBtn" >
            <property name="text" >
             <string>Bevel</string>
            </property>
            <property name="checked" >
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="miterJoinStyleBtn" >
            <property name="text" >
             <string>Miter</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="roundJoinStyleBtn" >
            <property name="text" >
             <string>Round</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="penColorBox" >
     <property name="title" >
      <string>Pen color</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
